docs/plugins/plugins.schema.json

```diff
@@ -12,10 +12,40 @@
           "github-repo": {
             "description": "GitHub repository",
             "type": "string"
+          },
+          "homepage": {
+            "description": "A website describing the plugin. If not present, we'll use the GitHub UI",
+            "type": "string"
+          },
+          "name": {
+            "description": "The value users of the plugin put in the 'name' field of their .aspect/cli/plugins.yaml when using this plugin",
+            "type": "string"
+          },
+          "maintainers": {
```

Comment:
note, I made this match the bazel-central-registry metadata.json schema just to reduce our overall cognitive load, may as well match the other place we publish stuff.

Reply:
nice; looks good